tests/migration/online_migration/zypper_migration.pm

```diff
@@ -196,4 +196,8 @@ sub post_fail_hook {
     $self->SUPER::post_fail_hook;
 }
 
+sub test_flags {
+    return {fatal => 1, milestone => 1};
```

Comment:
installbasetest.pm already set fatal, please drop it.

Reply:
Dropped 'fatal'.